YukonVaccinationVerifier/YukonVaccinationVerifier/BusinessGuidanceView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  BusinessGuidanceView.swift
+//  YukonVaccinationVerifier
+//
+//  Created by Mohamed Fawzy on 16/11/2021.
+//
+
+import Foundation
+import UIKit
+import SafariServices
+
+class BusinessGuidance {
+    
+    static func setupView(in presenter: UIViewController, links: [BusinessGuidanceLink]) {
+        guard let view = presenter.view else {
+            return
+        }
+        let backgroundView = UIView()
+        backgroundView.backgroundColor = UIColor(hexString: "#244C5A")
+        view.addSubview(backgroundView)
+        allignView(backgroundView,containerView: view, topMargin: nil, height: 85)
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 10
+        backgroundView.addSubview(stackView)
+        allignView(stackView, containerView: backgroundView, topMargin: 10, bottomMargin: nil)
+        backgroundView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        for link in links {
+            let button  = buttonLink(title: link.title)
+            button.addAction {
+                guard let urlString = link.url, let url = URL(string: urlString) else {
+                    return
+                }
+                let safariViewController = SFSafariViewController(url: url)
+            presenter.present(safariViewController, animated: true)
+            }
+            stackView.addArrangedSubview(button)
+        }
+    }
+    
+    private static func allignView(_ view: UIView, containerView: UIView, topMargin: CGFloat? = 0, leadingMargin: CGFloat? = 0, bottomMargin: CGFloat? = 0, trailingMargin: CGFloat? = 0, height: CGFloat? = nil) {
```

Comment:
Its better to define this on an extension for re-usability and also this isn't associated with what this class does.